README.md

```diff
@@ -286,6 +286,7 @@
 - [validate](https://github.com/endeveit/v-validate) - A simple library to validate strings in V.
 - [valval](https://github.com/taojy123/valval) - Web framework written in V, improved by vweb.
 - [vcurrency](https://github.com/mehtaarn000/vcurrency) - API wrapper (written in V) for [https://api.exchangeratesapi.io](https://api.exchangeratesapi.io).
+- [vest](https://github.com/alexferl/vest) - A V REST client.
```

Comment:
I think `A REST Client in V` would sound better

Reply:
Agreed, I changed it.